src/map/battle.cpp

```diff
@@ -7230,6 +7230,12 @@ static struct Damage initialize_weapon_data(struct block_list *src, struct block
 			case TK_TURNKICK:
 				wd.blewcount = 0;
 				break;
+			case SG_SUN_WARM:
+			case SG_MOON_WARM:
+			case SG_STAR_WARM:
+				// Knockback is random
```

Comment:
It would be better to clarify that there is a fixed part on the db and a random part here

Reply:
Comment improved.